### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by frisouilles**
### Description of the Change

Add some badges and description to lunch and contribute to this projects

### Possible Drawbacks

@KamiAmiga we need a banner for this project ! thank's

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,50 @@
-# ovh-icon-database-front
+# Ovh Icon Database Front
+
+![ovh-manager-telecom-banner]()
```
Comment (KamiAmiga) [inline README.md]: how about using this banner ? -> https://user-images.githubusercontent.com/3379410/27423240-3f944bc4-5731-11e7-87bb-3ff603aff8a7.png

Comment (frisouilles) [reply README.md]: Is it the banner for component, not for a complete ihm ? no ?

Comment (frisouilles) [reply README.md]: but if you thing it's OK. i'm Ok.

Comment (frisouilles) [reply README.md]: @KamiAmiga done

**Pull request merged**